v0/destinations/ga/transform.js

```diff
@@ -109,6 +109,39 @@ function getCustomParamsFromOldConfig(config) {
   return [dimensions, metrics];
 }
 
+// Function for processing pageviews
+function processPageViews(message, destination) {
+  let documentPath;
+  let documentUrl;
+  let hostname;
+  let { includeSearch } = destination.Config;
+  includeSearch = includeSearch || false;
+  if (message.properties) {
+    documentUrl = getFieldValueFromMessage(message, "GApageUrl");
+    let url;
+    if (documentUrl) {
+      try {
+        url = new URL(documentUrl);
+        hostname = url.hostname;
+        documentPath = url.pathname;
+        if (message.properties.search && includeSearch) {
```

Comment:
check search term in context.page as well? 
better to use getFieldValueFromMessage ?